config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -22,36 +22,26 @@ presets:
       secretKeyRef:
         name: clusterapi-provider-vsphere-ci-prow
         key: vsphere-password
-  - name: TARGET_VM_SSH
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: target-vm-ssh
-  - name: TARGET_VM_SSH_PUB
-    valueFrom:
-      secretKeyRef:
-        name: clusterapi-provider-vsphere-ci-prow
-        key: target-vm-ssh-pub
   volumeMounts:
-  - name: jumphost-key
-    mountPath: /root/ssh/.jumphost
-  - name: bootstrapper-key
-    mountPath: /root/ssh/.bootstrapper
+  - name: vpn-conf
+    mountPath: /vpn
```

Comment:
Hi @yastij,

The mount path needs to be `/root/.openvpn` since the OpenVPN config uses that as the path in its config:

```yaml
  prow.ovpn: |-
    client
    tls-client
    ca /root/.openvpn/ca.crt
    cert /root/.openvpn/client.crt
    key /root/.openvpn/client.key
    tls-crypt /root/.openvpn/tls.key
```

Reply:
Fixed